internal/migration_acceptance_tests/acceptance_test.go

```diff
@@ -42,7 +43,9 @@ type (
 	planFactory func(ctx context.Context, connPool sqldb.Queryable, tempDbFactory tempdb.Factory, newSchemaDDL []string, opts ...diff.PlanOpt) (diff.Plan, error)
 
 	acceptanceTestCase struct {
-		name         string
+		name string
+		// roles is a list of roles that should be created before the DDL is applied
+		roles        []string
```

Comment:
thought: I wonder if we should make it possible to define roles in schema somehow. Since they're global I figure that would erode the "you're only applying a single schema" paradigm we have, but it feels like a shame that roles have to be defined outside of pg-schema-diff.